Source/WebCore/Modules/web-locks/WebLockManager.cpp

```diff
@@ -370,4 +370,20 @@ bool WebLockManager::virtualHasPendingActivity() const
     return !m_pendingRequests.isEmpty() || !m_releasePromises.isEmpty();
 }
 
+void WebLockManager::suspend(ReasonForSuspension reason)
+{
+    // If we're being suspended due to page navigation or unload, force cleanup
```

Comment:
Comments in WebKit end with a period.

Reply:
Also the comment doesn't make a lot of sense, I wouldn't expect `suspend()` to get called on unload. `suspend()` gets called when you navigate away from the page and the page gets suspended in the back/forward cache (or when pausing in WebInspector debugging I think).

Reply:
Removed.